.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
+matrix:
+  allow_failures:
+    - python: pypy
+    - python: 3.7-dev
 python:
   - "2.6"
   - "2.7"
   - "3.4"
+  - "3.5"
```

Comment:
Up to this point, the versions need to be supported holistically is getting a bit too wild. I wonder what's the actual value of doing so? Shouldn't we spend more time on pushing new rather than lingering retrospectively?